build/env.sh

```diff
@@ -18,7 +18,7 @@ brave_env::set_brave_env() {
       esac
     else
       case ${0##*/} in
-      *.sh)
+      env.sh)
```

Comment:
this allows `build/env.sh` to be sourced from another script to do things like these:
```
. $src_dir/brave/build/env.sh
cd $BUILD_DIR
ninja -t targets all | grep -i '^gen/' | grep -E "\.(cpp|h|inc|cc)\:" | cut -d':' -f1 | xargs autoninja -k 0
```